src/components/ListItem.jsx

```diff
@@ -29,18 +29,42 @@ export function ListItem({ listToken, item }) {
 		}
 	}, [listToken, item]);
 
+	const getTextColor = () => {
+		if (item?.currentEstimate <= 7) {
+			return 'red';
+		} else if (item?.currentEstimate > 7 && item?.currentEstimate <= 30) {
+			return 'green';
+		} else if (item?.currentEstimate > 30 && item?.currentEstimate <= 60) {
+			return 'blue';
+		} else if (item?.currentEstimate > 60) {
+			return 'gray';
+		}
+	};
+
+	const getAria = () => {
+		if (item?.currentEstimate <= 7) {
+			return 'soon';
+		} else if (item?.currentEstimate > 7 && item?.currentEstimate <= 30) {
+			return 'kind of soon';
+		} else if (item?.currentEstimate > 30 && item?.currentEstimate <= 60) {
+			return 'not so soon';
+		} else if (item?.currentEstimate > 60) {
+			return 'inactive';
+		}
+	};
```

Comment:
Nice separating the utility function and then calling it at the point where you need it, it helps with code cleanness and readability👍